### PR TITLE
Ensure that loc comments in PDP's are immediately before their content

### PR DESCRIPTION
**Pull request by HowardWolosky**
The Loc team's parser expects the loc comments to be directly before the content
that the comment refers to.

### CODE REVIEW

File: Extensions/ConvertFrom-ExistingIapSubmission.ps1 [hunk 1]
```diff
@@ -206,7 +206,7 @@ function Add-Title
     $paramSet = @{
         "Element" = $elementNode;
         "Attribute" = @{ $script:LocIdAttribute = $script:LocIdFormat -f $elementName };
-        "Comment" = @($script:CommentFormat -f $maxChars, "IAP $elementName"; " [required] ")
+        "Comment" = @(" [required] "; $script:CommentFormat -f $maxChars, "IAP $elementName")
```
Comment (danbelcher-MSFT) [inline Extensions/ConvertFrom-ExistingIapSubmission.ps1]: Shouldn't this be a comma instead of semi-colon?

Comment (danbelcher-MSFT) [inline Extensions/ConvertFrom-ExistingIapSubmission.ps1]: Let's wrap the format operation in parentheses so we avoid the same situation we just ran into with Write-Log.  Here and elsewhere in this change (below)

Comment (HowardWolosky) [reply Extensions/ConvertFrom-ExistingIapSubmission.ps1]: Looks like both semi-colon _and_ comma are supported as array value separators...but I agree with you that we should be normalized on using comma.

Comment (HowardWolosky) [reply Extensions/ConvertFrom-ExistingIapSubmission.ps1]: agreed

**Pull request merged**